editor/src/clj/editor/code/shader.clj

```diff
@@ -88,6 +88,13 @@
                    :icon-class :script
                    :view-types [:code :default]
                    :view-opts glsl-opts}
+                  {:ext "cp"
+                   :language "glsl"
+                   :label "Compute Program"
```

Comment:
So the `.cp` is the "Compute Program"? What does that make the `.compute_program`?